shared/haml/user_header.haml

```diff
@@ -18,7 +18,7 @@
   should_show_create_menu = !level || level.try(:is_project_level)
 
 - if should_show_create_menu
-  .header_button.create_menu
+  %button.header_button.create_menu{tabindex: 0}
```

Comment:
I think in this case `tabindex` is redundant since the button should make it tab-able